index.js

```diff
@@ -2013,6 +2030,27 @@ module.exports = class Autobase extends ReadyResource {
     return false
   }
 
+  _checkLocalIndexerUpdate () {
+    if (!this.localWriter) return
```

Comment:
mb also check .localWriter.isActiveIndexer === .isIndexer and return to avoid the loop below in 99% of cases

Reply:
ok updated to skip if we are not expecting a removal, but i don't think the above check would work, cause when they are both false the event won't fire